src/mqt/predictor/rl/predictorenv.py

```
@@ -14,10 +14,11 @@
 import sys
 from typing import TYPE_CHECKING, Any
 
+from pytket._tket.passes import BasePass as TketBasePass  # noqa: PLC2701
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLC2701`)

Remove unused `noqa` directive

(RUF100)